src/json/any.cr

```diff
@@ -18,7 +18,7 @@
 # which return `nil` when the underlying value type won't match.
 struct JSON::Any
   # All possible JSON types.
-  alias Type = Nil | Bool | Int64 | Float64 | String | Array(Any) | Hash(String, Any)
+  alias Type = Nil | Bool | Int64 | UInt64 | Float64 | String | Array(Any) | Hash(String, Any)
```

Comment:
This can't be done. It's a breaking change. If someone had this type stored, they now need to also handle this.